src/components/modals/login-modal/login-content.less

```diff
@@ -0,0 +1,106 @@
+.ant-btn.ant-btn-primary.login-button {
```

Comment:
Same as above ^^^
We want to avoid overwriting AntD classes since they are global styles that may affect other components! If possible can you change this to not use AntD classes: i.e. styled-components or just applying a custom class?